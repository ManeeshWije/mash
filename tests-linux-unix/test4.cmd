ls | grep -v ^d | tr -s ' ' | tr ' '
