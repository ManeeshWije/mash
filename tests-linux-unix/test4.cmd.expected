getopt.c
grepped.output
Makefile.Darwin
Makefile.Linux
Makefile.Windows
mash
mash_main.c
mash_main.o
mash_run.c
mash_run.h
mash_run.o
mash_tokenize.c
mash_tokenize.h
mash_tokenize.o
numberlines.exe
os_defs.h
printsomething.exe
redir2.output
redir.output
runtests-linux-unix
runtests-windows.bat
tests-linux-unix
tests-windows
